bin/create-animal-db

```diff
@@ -16,7 +16,7 @@
 // with conflicts for the zebra doc
 
 
-const url = 'http://localhost:5984/';
+const url = 'http://tester:testerpass@localhost:5984/';
```

Comment:
I would like to change it to that. We always end up using the docker couchdb instance anyway. This is so much easier